### PR TITLE
added Sabrina's info under section A in aboutus.html, changed background color and font of all webpages

### PR DESCRIPTION
**Pull request by sabrinakundu**
I have edited files aboutus.html, main.css, index.html, and user-page.html. Most edits are in the aboutus.html file. Styling and CSS edits have been made in the other 3 files. In aboutus.html file, I have added my information under section A and deleted section D because we only have 3 teammates.

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -15,8 +15,12 @@
  */
 
 /* Prevent the page from becoming too wide. */
```
Comment (bandreasv) [inline src/main/webapp/css/main.css:17]: Seems like this comment is not in the wrong place. Please place it right above the piece of code that the comment is referring too.

File: src/main/webapp/css/main.css [hunk 2]
```diff
@@ -15,8 +15,12 @@
...
+.doc {
```
Comment (bandreasv) [inline src/main/webapp/css/main.css:18]: Add a comment as to what this is doing.

**Review by bandreasv — COMMENTED** [review]
Overall great job! Just minor comments to change.

**Comment by ldickeson** [comment]
Great job. I like what you do with the index.html page. Also good job on your aboutus section!

**Review by ldickeson — APPROVED** [review]
You did a good job being organized and adding in extra stuff!

File: src/main/webapp/css/main.css (hunk 1 above)
Comment (sabrinakundu) [reply src/main/webapp/css/main.css:17]: Ok, I fixed the error and created a new pull request.

File: src/main/webapp/css/main.css (hunk 2 above)
Comment (sabrinakundu) [reply src/main/webapp/css/main.css:18]: Ok, I made the change in the new pull request.

**Pull request merged**